src/scss/components/_donors.scss

```diff
@@ -1,29 +1,41 @@
-.donors_info {
-  @include grid;
-}
-
 .donors {
   position: relative;
   overflow: hidden;
 }
 
+.donors_info {
+  @include grid;
+
+  display: block;
+}
+
 .donors_image {
   width: 100%;
```

Comment:
Add `line-height: 0;` to this class in order to remove an extra space below the image

Reply:
Thanks for the comments, I already uploaded the improvements